wikipages/II.A.1 ml‐config.md

```diff
@@ -0,0 +1,326 @@
+This folder would contain various MarkLogic configuration details. These would control how resources like databases, roles, servers, etc. behave. Changes to any of the files under this folder would require the execution of the following command:
+
+```
+gradle mlDeploy
+```
+
+Smaller commands can be used to deploy specific sub-folders of this main folder. More info is available in the [Task Reference](TODO: task page to be built) page
```

Comment:
I'm starting to buy into the "Task Reference" page. Before, I was taking the output of "gradle tasks" and then formatting it for the Wiki. That was annoying to do every time a task was added. Instead, I think it can be like what you're suggesting - similar to the Property Reference, where it's a nice table, and some tasks can have links to other pages. It'll take some time to build the first page, but it'll be easy to update when new tasks are added.